pkg/apis/extensions.sk8s.io/v1/function.go

```diff
@@ -40,6 +40,8 @@ type FunctionSpec struct {
 	// +optional
 	Output string `json:"output,omitempty"`
 
+	IdleTimeout int `json:"idleTimeout,omitempty"`
```

Comment:
Needs to be pointer to int to be optional.
Everything exposed at k8s level ought to use int32(64) to guarantee portability.

Changing on merge

Reply:
Also, the unit should appear in the name, as described in https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#naming-conventions and https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#units
Changing to IdleTimeoutMs, let's reopen if this doesn't suit you

Reply:
Sounds good. I was considering whether it should be in *seconds*. This is probably fine for now, but let me know if you have an opinion.